spec/rails_helper.rb

```diff
@@ -9,6 +9,8 @@
 require File.join(File.dirname(__FILE__), 'support/valid_attribute')
 require File.join(File.dirname(__FILE__), 'support/factory_girl')
 require 'capybara/rspec'
+Dir[File.expand_path(File.join(File.dirname(__FILE__),'support','**','*.rb'))].each {|f| require f}
```

Comment:
Line is too long. [99/80]<br>Space missing after comma.<br>Space between { and | missing.<br>Space missing inside }.